.github/pull_request_template.md

```diff
@@ -2,13 +2,17 @@
 Thanks for contributing to Backpack :pray:
 
 Please include a description of the changes you are introducing and some screenshots if appropriate.
+
+Please ensure your pull request title is clear as it will be used to generate the changelog.
+
+Add `major`, `minor` or `patch` label depending on the change according to [semver](semver.org) or `skip-changelog` if the change shouldn't be added to the changelog (e.g. a change to a test or documentation)
 -->
 
 + [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)
 
 Remember to include the following changes:
-+ [ ] `UNRELEASED.md`
++ [ ] Component `README.md`
```

Comment:
unreleased gone! but i realised after we removed readme from here that we still have the readme for the component here.